src/views/darkroom.c

```diff
@@ -2234,7 +2234,7 @@ void gui_init(dt_view_t *self)
     ac = dt_action_define(sa, N_("raw overexposed"), N_("toggle"), dev->rawoverexposed.button, &dt_action_def_toggle);
     dt_shortcut_register(ac, 0, 0, GDK_KEY_o, GDK_SHIFT_MASK);
     gtk_widget_set_tooltip_text(dev->rawoverexposed.button,
-                                _("toggle raw over exposed indication\nright click for options"));
+                                _("toggle indication of raw overexposure\nright click for options"));
```

Comment:
As a native English speaker let me say

If I wrote this on a paper and turned it in to my English teacher they would "bleed" all over it (cross it out in red ink) and write "raw overexposure indicator" because that's what it is.  Or, if we are trying to say what it does then it would be "marks raw overexposed areas".  English favors direct phrasing instead of indirect.  

While I'm talking about English from a native speaker point of view I'm going to visit enlarge canvas.  Processing modules perform actions on an image, therefore using verbs in the names of modules makes sense to me.  If I want to crop an image I look for crop.  If I want to enlarge the canvas, then I would look for enlarge canvas.  Canvas enlargement sounds an awful lot like all those spam emails I get advertising another 5 letter word ending in 's' enlargement.

I'm not trying to criticize.  I understand what you did and why you did it and I can live with it.  But, perhaps you should run English wording changes past @paperdigits or @elstoc to make sure they convey the proper meaning in the best way possible for English speakers.  If they can't do it, then tag me and I'll look at it and give you my opinion.

Reply:
I agree with everything you've said. I don't think there was anything wrong with "toggle raw overexposed indication", though maybe "raw overexposed overlay" might have been an improvement? Similarly "enlarge canvas" is a better module name